app/views/visual_models/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_with(model: visual_model) do |form| %>
+  <% if visual_model.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(visual_model.errors.count, "error") %> prohibited this visual_model from being saved:</h2>
+
+      <ul>
+        <% visual_model.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
Don't we usually raise flash errors if a form doesn't save? That seems to be how it works on Prefs and New obs.

Reply:
Did it for new/create.  Haven't dug into edit/update yet.

Reply:
Did it for new/create.  Haven't dug into edit/update yet.

Reply:
Done.